### PR TITLE
add Onion and I2P links to footer

### PR DESCRIPTION
**Pull request by HardenedSteel**
image

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -36,6 +36,24 @@ rssLimit = 20
...
     url = "https://docs.featherwallet.org/guides/report-an-issue"
     weight = 6
+  [[menu.link]]
+    name = "Clearnet"
+    pre = "clearnet"
+    url = "http://featherwallet.org/"
+    weight = 1
+  [[menu.link]]
+    name = "Onion Service"
+    pre = "onionservice"
```
Comment (nahuhh) [inline config.toml:46]:
```suggestion
    name = "Onion"
    pre = "onionservice"
```

File: config.toml [hunk 2]
```diff
@@ -36,6 +36,24 @@ rssLimit = 20
...
+    params.icon = "<img src='/img/onion-tor.svg'>"
+    url = "http://featherdvtpi7ckdbkb2yxjfwx3oyvr3xjz3oo4rszylfzjdg6pbm3id.onion/"
+    weight = 2
+  [[menu.link]]
+    name = "I2P Site"
+    pre = "i2psite"
```
Comment (nahuhh) [inline config.toml:52]:
```suggestion
    name = "I2P"
    pre = "i2psite"
```

**Review by nahuhh — COMMENTED** [review]
id prefer to avoid fragmented descriptors like clearnet(none)/onion(service)/i2p(site)

File: config.toml [hunk 3]
```diff
@@ -36,6 +36,24 @@ rssLimit = 20
     pre = "help"
     url = "https://docs.featherwallet.org/guides/report-an-issue"
     weight = 6
+  [[menu.link]]
+    name = "Clearnet"
+    pre = "clearnet"
+    url = "http://featherwallet.org/"
```
Comment (tobtoht) [inline config.toml:42]:
```suggestion
    url = "https://featherwallet.org/"
```

**Pull request merged**